nemo_rl/data/datasets/utils.py

```diff
@@ -72,7 +72,13 @@ def load_dataset_from_path(data_path: str, data_split: Optional[str] = "train"):
     if suffix in [".json", ".jsonl"]:
         raw_dataset = load_dataset("json", data_files=data_path)
     else:
-        raw_dataset = load_dataset(data_path)
+        try:
+            raw_dataset = load_dataset(data_path)
+        except ValueError as e:
+            if "load_from_disk" in str(e):
+                raw_dataset = load_from_disk(data_path)
+            else:
+                raise
```

Comment:
It's better to show the error.
```suggestion
                raise e
```